### ISSUE TITLE
[Accessbility]A11y_Java Tooling for IntelliJ/Eclipse - New Features_Remote Debugging-Debugger_Screen Reader: Jaws is not reading the tab names "Frames" & "Variables"

### ISSUE THREAD

**Issue by InduPriya1805** [microsoft/azure-tools-for-java#3633]
**GitHub Tags:** #A11y_JavaToolingforIntelliJ/Eclipse–NewFeature; #A11yMAS; #A11yTCS; #C+E-H; #DesktopApp; #Win10; #JavatoolingforIntelliJ/Eclipse; #ScreenReader; #JAWS; #MAS1.3.1; 

[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier

**Environment Details:**
Download and Install IntelliJ IDEA from https://www.jetbrains.com/idea/download/#section=windows
IntelliJ IDEA 2019.2.1
Build #IU-192.6262.58

**OS Details:**
OS: Windows (Version 1903, Build: 18362.113)

**MAS Violated:** 1.3.1

**Repro Steps:**
1)Turn on screen reader (Jaws) 
2)Launch IntelliJ IDEA
3)Tab till create new project-->Azure Spark/HDInsight-->Spark project with Samples(Scala) 
4)Tab till next button and press enter
5)Fill all the details correctly and Tab till finish button and press enter
6)Tab till <Project Name>-->src-->main-->scala-->sample-->SparkCore_wasbloTest and open it
7)Tab till Edit Configurations menu(select the icon in the upper-right corner)
8)Navigate to Advanced Configuration (Remote Debugging) section and give the required details & press Ok button
9)Sent some breakpoints in SparkCore_wasbloTest code and tab till sparkjobdebug and press enter
10)Press cntrl+tab to navigate to Debug Remote Spark Job in Cluster section
11)Tab till frames and variables tab and verify jaws is reading the tabs names or not

**Actual Result:**
Jaws is not reading the tab names "Frames" & "Variables"

**Expected Result:**
Jaws should  read the tab names "Frames" & "Variables"

**User Impact:**
Screen Reader users will not able to get to know about the tab names if jaws does not read them

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

**Attachment:**
[Bug_3633_jaws not reading tab names frames and variables.pptx](https://github.com/microsoft/azure-tools-for-java/files/3656020/Bug_3633_jaws.not.reading.tab.names.frames.and.variables.pptx)

**Comment by t-rufang** [comment]
Note: When focus is at `Frame` tab, screen reader is not reading the tab name
![image](https://user-images.githubusercontent.com/32627233/65664023-f7334b80-e06a-11e9-8757-3483e7b0b106.png)

**Comment by Alexandria-R** [comment]
This has been approved to close by SME.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749729
